### PR TITLE
SSB: Restyle code using ${}

### PR DESCRIPTION
**Pull request by DeathlyPlays**
Made it look a lot neater, etc (also fixed every known typo)

### CODE REVIEW

File: wavelength-plugins/ssb.js [hunk 1]
```diff
@@ -605,419 +609,435 @@ try {
...
 		],
-		forceupdate: 'validate',
-		validateall: 'validate',
+
+		forceupdate: "validate",
+		validateall: "validate",
 		validate: function (target, room, user, connection, cmd, message) {
-			if (!this.can('ssb')) return;
-			if (!target && toId(cmd) !== 'validateall') return this.parse('/help ssb validate');
+			if (!this.can("ssbffa")) return;
```
Comment (Volco) [inline wavelength-plugins/ssb.js]: Pretty sure the config file permission is ssb not ssbffa. need @HoeenCoder's confirmation

Comment (DeathlyPlays) [reply wavelength-plugins/ssb.js]: oh it is my bad :x

**Comment by Volco** [comment]
@DeathlyPlays all tested, all works, no errors? (just asking to be sure lol)

**Comment by DeathlyPlays** [comment]
Yeah, as far as I know, but there has been a recent bug before I did this where I noticed setting regular moves fails @Volco but that wasn't caused by this

**Comment by Volco** [comment]
think you can fix that and add it to this PR please? @DeathlyPlays

**Comment by DeathlyPlays** [comment]
I'd have to look into it

**Comment by Volco** [comment]
okay @DeathlyPlays then i'll just merge this and if you do look into it you can make a new pr for it (if not i'll look into it)

**Comment by DeathlyPlays** [comment]
ok

**Pull request merged**